src/pages/Swap.tsx

```diff
@@ -323,21 +346,33 @@ function Swap(): ReactElement {
           formStateArg.to.tokenIndex,
           amountToGive,
         )
+        console.log("direct", { amountToReceive })
       } else if (
         formStateArg.swapType === SWAP_TYPES.SYNTH_TO_SYNTH &&
         snxEchangeRatesContract != null
       ) {
+        const fromTokenSymbol = tokens[formStateArg.from.address]?.symbol
```

Comment:
why not just `formStateArg.from.symbol`?

Reply:
I rushed this part

Reply:
keying off addr here now